tests/model/DataExtensionTest.php

```diff
@@ -159,13 +159,20 @@ public function testExtensionCanBeAppliedToDataObject() {
 		$do = new DataObject();
 		$mo = new DataExtensionTest_MyObject();
 
-		$this->assertTrue($do->hasMethod('testMethodApplied'));
 		$this->assertTrue($mo->hasMethod('testMethodApplied'));
+		$this->assertTrue($do->hasMethod('testMethodApplied'));
```

Comment:
No. None. I think I temporarily removed the DO, to see if that caused an error in my initial test, and pasted it back on the wrong line, without checking if it was the right line.
Sorry